<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>conftool_import</class>
 <widget class="QDialog" name="conftool_import">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="bOK">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>590</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="uiTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>450</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
       <string>Database Run Configurations</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Filter by Name</string>
   </property>
   <widget class="QLineEdit" name="tfConfigName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>511</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>531</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>List of available configurations</string>
   </property>
   <widget class="QListView" name="lvConfigurationList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>511</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="bRefreshList">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>480</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>bOK</zorder>
  <zorder>uiTitle</zorder>
  <zorder>groupBox</zorder>
  <zorder>bRefreshList</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bOK</sender>
   <signal>accepted()</signal>
   <receiver>conftool_import</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bOK</sender>
   <signal>rejected()</signal>
   <receiver>conftool_import</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
